banyand/metadata/schema/data/index_rule_binding.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Now we are using `json` instead of `prototext` files?

Reply:
`json` is supported by IDE widely and properly. That means we can edit these metadata more easily than `prototext`.

BTW, once `metadata` management is ready, we will remove these files finally.

Reply:
Make sense to me